README.md

```diff
@@ -0,0 +1,37 @@
+# ElasticSearch on Rancher
+
+> Rancher configuration for running an Elasticsearch cluster
+
+## Services deployed:
+
```

Comment:
The rancher-config file has and additional service (`es-sysctl`) not mentioned here.

Reply:
I'll add it.